test/e2e/jobs/jobs.test.ts

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { GenericContainer, Network, StartedNetwork, StartedTestContainer, TestContainers, Wait } from "testcontainers";
+// import { LogWaitStrategy } from "testcontaineclienrs/build/wait-strategies/log-wait-strategy";
+import { CommunicationProtocolEnum, DaprClient, DaprServer } from "../../../src";
+// import { AbstractWaitStrategy } from "testcontainers/build/wait-strategies/wait-strategy";
+
+jest.setTimeout(40 * 1000);
```

Comment:
When working with testcontainers these timeouts are usually much longer, e.g., 120_000 or 180_000.

The testcontainers default wait is 60_000, so you want jest to wait at least that long.